File: fizzbuzz/fizzBuzz.pl
#!/usr/bin/perl

# Fizzbuzz in PERL
# by Marshall Ehlinger

use strict;
use warnings;

my $MAX = 100;

for (my $i = 0; $i <= $MAX; $i++) {
	my $line = "";
	if ($i % 3 == 0) { $line .= "fizz"; }
	if ($i % 5 == 0) { $line .= "buzz"; }
	if ($line eq "") { $line = $i; }
	print("$line\n");
}

File: fizzbuzz/FizzBuzz.java
// FizzBuzz in Java
// by Marshall Ehlinger

public class FizzBuzz {

    public static void main (String[] args) {
        // Test Program
        final int MAX = 100;
        fizzBuzz(MAX);
    }

    public static void fizzBuzz(int max) {
        for (int i = 1; i <= max; i++) {
            StringBuilder line = new StringBuilder();
            if (i % 3 == 0) {
                line.append("fizz");
            }
            if (i % 5 == 0) {
                line.append("buzz");
            }
            if (line.length() == 0) {
                line.append(i);
            }
            System.out.println(line);
        }
    }
}